12_Highest_Divisible_Triangle.py

```diff
@@ -0,0 +1,48 @@
+import math
+#not using this one
+def get_triangle_number(nth_term):
+    i = nth_term
+    current_sum = 0
+    while(i > 0):
+        current_sum = current_sum + i
+        i -= 1
+    return current_sum
+
+def find_num_divisors(num):
+    divisor = 1
+    num_divisors = 0
+    #if num%2 == 1: # is odd
+    while (divisor <= math.sqrt(num)):
+        if num%divisor==0:
+
+            if num/divisor == divisor:
+                num_divisors += 1
+            else:
```

Comment:
Can you also have e.g. 27 / 3 = 9, which is yet again divisible by 3? So not necessarily always a +2, but maybe more?

Reply:
Cant a multiple only be made by either 2 different numbers or the same number squared?